common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala

```diff
@@ -103,6 +113,34 @@ class SecondarySchemaLoader[CT, CV](conn: Connection, dsLogger: Logger[CT, CV],
     }
   }
 
+  /**
+    * To suppress index create
+    *   INSERT into index_directives(dataset_system_id, field_name, directives) VALUES (${datasetSystemId}, ${fieldName}, '{"enable": false}')
+    *      (fieldName can be null to apply to all columns)
+    * To trigger reindex
+    *   curl -v  -X POST -H 'Content-Type: application/json' 'http://localhost:6010/dataset/${resourceName}/!secondary-reindex' --data '{}'
+    */
+  private def shouldCreateIndex(stmt: PreparedStatement, datasetId: DatasetId, fieldName: Option[ColumnName]): Boolean = {
+    fieldName.flatMap { field =>
+      stmt.setLong(1, datasetId.underlying)
+      stmt.setString(2, field.name)
```

Comment:
Database-level comparisons should be done with `field.casefolded` for case-insensitivity.  There ought to be a column on the table called `field_name_casefolded` or somesuch for that.

Reply:
ok